manifests/x/Xidicone/ZeusIDE/3.98y/Xidicone.ZeusIDE.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Xidicone.ZeusIDE
+PackageVersion: 3.98y
+InstallerSwitches:
+  Silent: /verysilent
+  SilentWithProgress: /silent

```

Comment:
It's failing because of this selection that has to be made before the installation starts.
![image](https://user-images.githubusercontent.com/52826317/123636372-11d8bc80-d81d-11eb-9569-1984f4972855.png)

This should do it. It will automatically choose user install (this is recommended)
```suggestion
  SilentWithProgress: /silent
  Custom: /CURRENTUSER
```